gamemodes/terrortown/gamemode/shared/sh_lang.lua

```diff
@@ -112,7 +112,7 @@ if SERVER then
 
 	---
 	-- @realm server
-	local cv_ttt_lang_serverdefault = CreateConVar("ttt_lang_serverdefault", "english", FCVAR_ARCHIVE)
+	local cv_ttt_lang_serverdefault = CreateConVar("ttt_lang_serverdefault", "en", FCVAR_ARCHIVE)
```

Comment:
do we have to add a coverter here so the convar with "english" is changed to "en"?

Reply:
As discussed on discord this isn't a problem at all.